test/functional/forall/atomic-basic/test-forall-atomic-basic.cpp.in

```diff
@@ -33,7 +38,7 @@ using @ATOMIC_BACKEND@ForallAtomicBasicTypes =
   Test< camp::cartesian_product<@ATOMIC_BACKEND@ForallAtomicExecPols,
                                 @ATOMIC_BACKEND@AtomicPols,
                                 @ATOMIC_BACKEND@ResourceList,
-                                AtomicSegmentList,
```

Comment:
Ah, I see why you did this. At the time, I thought that the combination of segment + index type would give us the most flexibility. As long as we're not going to add more segment types, this should be better. Thanks for digging into this!